test/test_srv_polling.py

```diff
@@ -230,6 +230,50 @@ def erroring_seedlist():
             raise ConfigurationError
         self._test_recover_from_initial(erroring_seedlist)
 
+    def test_10_all_dns_selected(self):
+        response = [("localhost.test.build.10gen.cc", 27017),
+                    ("localhost.test.build.10gen.cc", 27017),
```

Comment:
27017 appears twice here. Is that a bug?

Reply:
https://github.com/mongodb/specifications/blob/master/source/polling-srv-records-for-mongos-discovery/tests/README.rst#configuring-the-test-environment So here it says that you should have two with 27017 but they both resolve to the same thing so I'm not sure why. I was just following exactly what the test said.

Reply:
The second 27017 address is for a different SRV record (test3.test.build.10gen.cc). There should only be one 27017 in this test.